src/drafts/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -254,6 +276,34 @@ const MarkdownEditor = forwardRef<MarkdownEditorHandle, MarkdownEditorProps>(
       }
     )
 
+    useEffect(() => {
+      if (view === 'preview') {
+        editorsInPreviewMode.push(id)
+
+        const handler = (e: KeyboardEvent) => {
+          if (
+            !e.defaultPrevented &&
+            editorsInPreviewMode.at(-1) === id &&
+            isModifierKey(e) &&
+            e.shiftKey &&
+            e.key === 'p'
+          ) {
+            setView?.('edit')
+            requestAnimationFrame(() => inputRef.current?.focus())
```

Comment:
On second thought I'm pretty sure this should be `setTimeout`
```suggestion
            setTimeout => inputRef.current?.focus())
```

Reply:
This is what I get for trying to make this fix from my phone 🤦